po/cs.po

```diff
@@ -0,0 +1,644 @@
+msgid "Add more tools"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:112
+msgid "Make tools popover more powerful"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:113
+msgid "Redesign icon"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:114
+msgid "General improvements"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:119
+msgid "Add tools for replacement and removing"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.appdata.xml.in:122
+msgid "Application release"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.gschema.xml:11
+msgid "Color scheme"
+msgstr "Barevné schéma"
+
+#: data/com.github.liferooter.textpieces.gschema.xml:12
+msgid "Which color scheme to use for application interface."
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.gschema.xml:18
+msgid "Font name"
+msgstr "Jméno fontu"
+
+#: data/com.github.liferooter.textpieces.gschema.xml:19
+msgid "Name of font used in editor"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.gschema.xml:25
+msgid "Wrap lines"
+msgstr "Zarovnání řádků"
+
+#: data/com.github.liferooter.textpieces.gschema.xml:26
+msgid "Whether to wrap long lines"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.gschema.xml:32
+msgid "Tabs to spaces"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.gschema.xml:33
+msgid "Whether to insert spaces instead of tab"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.gschema.xml:39
+msgid "Spaces in tab"
+msgstr ""
+
+#: data/com.github.liferooter.textpieces.gschema.xml:40
+msgid "Tab width in spaces"
+msgstr ""
+
+#: src/NewToolPage.vala:129
+#, c-format
+msgid "Error occured: %s"
+msgstr "Došlo k chybě: %s"
+
+#: src/Preferences.vala:135
+msgid "Select font"
+msgstr "Vybrat font"
+
+#: src/ToolSettings.vala:198
+#, c-format
+msgid "Argument %d"
+msgstr "Argument %d"
+
+#: src/Window.vala:49 resources/ui/ShortcutsWindow.ui:68
+msgid "Select tool"
+msgstr "Vybrat nástroj"
+
+#: src/Window.vala:315 src/Window.vala:318
+msgid "Gleb Smirnov <glebsmirnov0708@gmail.com>"
+msgstr "Gleb Smirnov <glebsmirnov0708@gmail.com>"
+
+#: src/Window.vala:317
+msgid "Tobias Bernard https://tobiasbernard.com"
+msgstr "Tobias Bernard https://tobiasbernard.com"
+
+#: src/Window.vala:330
+msgid "translator-credits"
+msgstr "vikdevelop <super-vik1@protonmail.com>"
+
+#: src/Window.vala:350
+msgid "Text is copied to clipboard"
+msgstr "Text je zkopírován do schránky"
+
+#: src/Window.vala:359 resources/ui/Window.ui:114
+msgid "Save to File"
+msgstr "Uložit do souboru"
+
+#: src/Window.vala:388
+#, c-format
+msgid "Can't save to file: %s"
+msgstr "Není možné uložit do souboru: %s"
+
+#: src/Window.vala:404 resources/ui/Window.ui:121
+msgid "Load from File"
+msgstr "Načíst ze souboru"
+
+#: src/Window.vala:437
+#, c-format
+msgid "Can't load from file: %s"
+msgstr "Nelze načíst ze souboru: %s"
+
+#: resources/ui/CustomToolPage.ui:49
+msgid "D_elete"
+msgstr "Odstranit_"
```

Comment:
Underline should be written before the letter that you want to use as mnemonic accelerator (e.g. button with label "D_elete" can be acivated by pressing `Alt + E`). Your use of underline is incorrect.